packages/synapse-react-client/src/components/FeaturedDataTabs/FacetPlotsCardGrid.tsx

```diff
@@ -0,0 +1,109 @@
+import { Box, BoxProps, styled } from '@mui/material'
+import { StyledComponent } from '@emotion/styled'
+
+export const FACET_PLOTS_CARD_TITLE_CONTAINER_CLASSNAME =
+  'FacetPlotsCardTitleContainer'
+
+export const FACET_PLOTS_CARD_PLOT_CONTAINER_CLASSNAME =
+  'FacetPlotsCardPlotContainer'
+
+export const FacetPlotsCardGridContainer: StyledComponent<BoxProps> = styled(
+  Box,
+  {
+    label: 'FacetPlotsCardGridContainer',
+  },
+)(({ theme }) => ({
+  [theme.breakpoints.down('md')]: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: theme.spacing(2),
+  },
+  [theme.breakpoints.up('md')]: {
+    /* 
+    The grid is broken up into cards which have subcomponents that should be aligned
+    Ideally we could use CSS Subgrid, but it is not widely supported as of the time of writing
+    See https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_grid_layout/Subgrid
+
+    Instead we will use grid with `display: contents`, and style each section to look like a card.
+     */
+    display: 'grid',
+    gridTemplateRows: '[title] auto [plot0] auto',
+    gridTemplateColumns: '[col1] 32% [col2] 32% [col3] 32%',
+    columnGap: '2%',
+
+    '.FacetPlotsCard': {
+      display: 'contents',
```

Comment:
Display `contents` basically moves everything up one level, and any styles on this element are not rendered, even though it remains in the DOM.